Unable to run e2e tests with Node 24

### Build tool

Vite

### Where do you see the problem?

- [ ] In the browser
- [x] In the terminal

### Describe the bug

I'm on MacOS Tahoe 26.4.1 and Node v24.15.0.
Tried clearing cache, re-installing, rebooting etc.

Upgrading Chokidar solves this issue for me. (Will create a PR).


```
 RUN  v0.28.5 /Users/sskenderovic/OpenSource/chrome-extension-tools/packages/vite-plugin

 ❯ tests/e2e/mv3-vite-react-content-script-seq-hmr/vite-serve.test.ts (1)
   ✓ crx page update on hmr
     ⠙ [ afterEach ]
/Users/sskenderovic/OpenSource/chrome-extension-tools/packages/vite-plugin:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @crxjs/vite-plugin@2.4.0 test:run:e2e: `vitest --run --mode e2e`
Command failed with signal "SIGABRT"
```

### Reproduction

`pnpm --filter '@crxjs/vite-plugin' test:run:e2e`

### Logs

```shell

```

### System Info

```shell
System:
    OS: macOS 26.4.1
    CPU: (12) arm64 Apple M2 Pro
    Memory: 1.84 GB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 24.15.0 - /Users/sskenderovic/.nvm/versions/node/v24.15.0/bin/node
    npm: 11.12.1 - /Users/sskenderovic/.nvm/versions/node/v24.15.0/bin/npm
    pnpm: 10.11.1 - /Users/sskenderovic/OpenSource/chrome-extension-tools/node_modules/.bin/pnpm
    bun: 1.1.21 - /opt/homebrew/bin/bun
    Deno: 1.45.2 - /opt/homebrew/bin/deno
  Browsers:
    Brave Browser: 147.1.89.145
    Chrome: 147.0.7727.138
    Chrome Canary: 149.0.7816.0
    Edge: 147.0.3912.98
    Firefox: 150.0.1
    Safari: 26.4
```

### Severity

blocking an upgrade

--- a/.changeset/upgrade-chokidar.md
+++ b/.changeset/upgrade-chokidar.md
@@ -1,0 +1,5 @@
+---
+"@crxjs/vite-plugin": patch
+---
+
+chore: upgrade chokidar to 5.0.0

--- a/packages/vite-plugin/package.json
+++ b/packages/vite-plugin/package.json
@@ -111,7 +111,7 @@
     "@typescript-eslint/parser": "5.41.0",
     "@vitejs/plugin-react": "^2.2.0",
     "@vitejs/plugin-vue": "3.2.0",
-    "chokidar": "^3.5.3",
+    "chokidar": "^5.0.0",
     "esbuild": "0.17.14",
     "esbuild-runner": "2.2.2",
     "eslint": "8.43.0",

--- a/packages/vite-plugin/tests/runners.ts
+++ b/packages/vite-plugin/tests/runners.ts
@@ -160,7 +160,7 @@ export async function serve(dirname: string): Promise<ServeTestResult> {
   await allFilesReady()
   debug('bundle end')
 
-  const outDirSettle$ = fromEvent(watch(outDir), 'all').pipe(
+  const outDirSettle$ = fromEvent(watch(outDir) as unknown as NodeJS.EventEmitter, 'all').pipe(
     startWith(null),
     map((x, i) => i),
     // debounce relies on the Date object

